src/models/AuxiliaryChain.ts

```diff
@@ -0,0 +1,94 @@
+import BigNumber from 'bignumber.js';
+import Comparable from '../observer/Comparable';
+
+/**
+ * Represents AuxiliaryChain model object.
+ */
+export default class AuxiliaryChain extends Comparable<AuxiliaryChain> {
```

Comment:
I thought about this. 
I think in all/most cases we will get model object from database and then specific field will be passed for the update. It's always recommended to fetch the model object to make sure record exists before update instead of firing a direct update which can fail in case primaryKey doesn't exist.
e.g.
```
const auxiliaryChain = await config.repos.auxiliaryChainRepository.get(
      auxiliaryChain.chainId,
    );
auxiliaryChain.lastProcessedBlockNumber = new BigNumber('20');
await config.repos.auxiliaryChainRepository.save(
      auxiliaryChain,
    );
```

Reply:
If it does not exist, new entry will be created in case of calling save.

We've agreed that during an update, only defined props will be updated and it would not touch non-defined ones. Again, here we should align with stake request model and repository implementation. I am open for a future discussions, however, as ticket is about alignment, we should follow that path. I am open to discuss it more, however, believe it should be done outside of this ticket. What do you think?

Reply:
An advantage I see on updating partial fields on complete model object is that db `commit` message is fired. Because of this we will be able to handle hooks/triggers. 
http://docs.sequelizejs.com/manual/hooks.html

I have faced this issue with mysql db. I assume it works similarly on postgres/sequelite. With partial database updates(e.g. `UPDATE lastProcessedBlockNumber where  chainId =1`) db `commit` message it not fired.

However we are not using hooks now. We can discuss later.
I have made the changes for now.